components/tooltip/package.json

```diff
@@ -17,16 +17,16 @@
   },
   "peerDependencies": {
     "@spectrum-css/icon": "^3.0.21",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^3.0.0"
   },
   "devDependencies": {
     "@spectrum-css/component-builder": "^3.2.0",
     "@spectrum-css/icon": "^3.0.23",
-    "@spectrum-css/vars": "^8.0.0",
+    "@spectrum-css/tokens": "^3.0.0",
```

Comment:
Would you mind bumping this (and the peerDependency, too) to `^4.0.0`?

Reply:
✔ Done